docs/devel/cherry-picks.md

```diff
@@ -42,14 +42,12 @@ depending on the point in the release cycle.
 
 1. Cherrypicks are [managed with labels and milestones]
 (pull-requests.md#release-notes)
-1. All label/milestone accounting happens on PRs on master. There's nothing to
-do on PRs targeted to the release branches.
-1. When you want a PR to be merged to the release branch, make the following
-label changes to the **master** branch PR:
-  * Remove release-note-label-needed
-  * Add an appropriate release-note-(!label-needed) label
-  * Add an appropriate milestone
-  * Add the `cherrypick-candidate` label
+1. To get a PR merged to the release branch, ensure the following labels are
+   on the **master** branch PR:
```

Comment:
Clarify this wording a little bit? 

"The PR containing the commits that will be merged into the master branch"?